apps/condo/domains/common/components/HeaderActions.tsx

```diff
@@ -0,0 +1,74 @@
+import { useIntl } from '@core/next/intl'
+import { LinkWithIcon } from './LinkWithIcon'
+import { ArrowLeftOutlined } from '@ant-design/icons'
+import { colors } from '../constants/style'
+import React, { useContext } from 'react'
+import { MessageDescriptor } from '@formatjs/intl/src/types'
+import Router, { useRouter } from 'next/router'
+import get from 'lodash/get'
+import { Space, Typography } from 'antd'
+import { AuthLayoutContext } from '@condo/domains/user/components/containers/AuthLayout'
+import { Button } from './Button'
+
+interface IReturnBackHeaderActionProps {
+    descriptor: MessageDescriptor
+    path: ((id: string) => string) | string
+}
+
+interface ITitleHeaderActionProps {
+    descriptor: MessageDescriptor
+}
+
+interface IRightButtonHeaderActionProps {
+    descriptor: MessageDescriptor
+    path: string
+}
+
+export const ReturnBackHeaderAction: React.FC<IReturnBackHeaderActionProps> = (props) => {
```

Comment:
Why pass descriptor as a prop? Wouldnt it be more readable to pass the string to the prop? 

Like `<ReturnBackHeaderAction title='matvey'/>`